pyogrio/_geometry.pxd

```diff
@@ -1,2 +1,2 @@
 cdef str get_geometry_type(void *ogr_layer)
-cdef int get_geometry_type_code(str geometry_type)
\ No newline at end of file
+cdef int get_geometry_type_code(str geometry_type) except *
```

Comment:
I don't see that these were exercised here; was this an existing bug?

Reply:
Yes, I added a test where I am passing a nonsense geometry type string, and currently the `get_geometry_type_code` was not properly raising an error for that.